### PR TITLE
Add 'about' text

### PR DESCRIPTION
**Pull request by saschagrunert**
Added a friendly "about" text to the page. We would still need to create the survey and add the link to it.

### CODE REVIEW

File: src/app/options/options.component.ts [hunk 1]
```diff
@@ -100,7 +100,37 @@ export class OptionsComponent implements OnInit {
...
+            further regarding your personal needs. In order to achive this goal
+            we prepared a survey and we would appreciate if you could take your
+            time to fill it out and help us to improve.
+        </p>
+        <p>
+            <a href="#" taget="_bank">
+                👉 Help us and participate in the survey.
+            </a>
+        </p>
+            We target to evaluate the results of the survey at the end of the
```
Comment (onyiny-ang) [inline src/app/options/options.component.ts]: We aim to. . .

File: src/app/options/options.component.ts [hunk 2]
```diff
@@ -100,7 +100,37 @@ export class OptionsComponent implements OnInit {
...
+            v1.16 release cycle on the official Kubernetes
+            <a href="https://groups.google.com/forum/#!forum/kubernetes-sig-release"
+               target="_bank">SIG release mailling list</a>.
+        </p>
+        <p>
+            Do you want to contribute or give further feedback? Great, feel
+            free to open up a pull request or issue within the
```
Comment (onyiny-ang) [inline src/app/options/options.component.ts]: . . .open up a pull request or issue in the. . .

File: src/app/options/options.component.ts [hunk 3]
```diff
@@ -100,7 +100,37 @@ export class OptionsComponent implements OnInit {
...
+            Thank you for giving the official Kubernetes Release Notes website
+            a try! The project aims to provide you with the best possible user
+            experience when it comes to finding out what changed in Kubernetes.
+        </p>
+        <p>
+            We would love to get your feedback about this website, especially
+            in terms of usability, look and feel and how to improve it even
+            further regarding your personal needs. In order to achive this goal
+            we prepared a survey and we would appreciate if you could take your
+            time to fill it out and help us to improve.
```
Comment (onyiny-ang) [inline src/app/options/options.component.ts]: This paragraph is grammatically fine but a bit awkward at times. I would suggest the following:

```
 We would love feedback on your experience using this website. We are particularly interested in:

 - how easy it is to use and find what you are looking for

 - the look and feel 

 - how to improve it further for your specific use case

We would appreciate it if you could take a few minutes to fill out our survey and help us to improve!
```

File: src/app/options/options.component.ts (hunk 1 above)
Comment (saschagrunert) [reply src/app/options/options.component.ts]: Fixed 👍

File: src/app/options/options.component.ts (hunk 3 above)
Comment (saschagrunert) [reply src/app/options/options.component.ts]: Okay, I changed it regarding your suggestion.

File: src/app/options/options.component.ts (hunk 2 above)
Comment (saschagrunert) [reply src/app/options/options.component.ts]: Fixed :)

**Comment by saschagrunert** [comment]
Thanks for the review! I changed the mentioned issues. PTAL 😇

**Comment by jeefy** [comment]
We should probably change the survey for something like Google Forms so we can share/change ownership easily. Not going to block on this, but wanna sort it out before release tomorrow

/approve
/lgtm

**Pull request merged**